tools/depends/target/dummy-libxbmc/dummy-libxbmc.c

```diff
@@ -2,3 +2,8 @@ void xbmc_dummy_function()
 {
   return;
 }
+
+int main()
```

Comment:
What is the reason for this? It shouldn't be needed in a shared library

Reply:
Tvos complains of no main function when linking the pythonmodules. As we don't have Kodi yet for its main function, I've shoe horned it into this. It's definitely required for tvos, but where we do it is up for debate I guess.

Reply:
please add a comment, so later generations know why it's there ;)